src/aind_behavior_curriculum/task.py

```diff
@@ -38,3 +38,9 @@ class Task(AindBehaviorModel):
     task_parameters: TaskParameters = Field(
         ..., description=TaskParameters.__doc__.strip(), validate_default=True
     )
+    version: Optional[str] = Field(
+        default=None,
+        pattern=r"^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$",  # noqa: E501
```

Comment:
make this pattern a module-level global? e.g. `SEMVER_PATTERN=blah` at the top level

Reply:
Fixed by [379aa31](https://github.com/AllenNeuralDynamics/aind-behavior-curriculum/pull/40/commits/379aa3188228e7e46b916a3045e2fb3264f12daa)